pkgs/development/tools/build-managers/cmake/search-path.patch

```diff
@@ -39,34 +40,11 @@ diff -ur cmake-3.9.1/Modules/Platform/UnixPaths.cmake cmake-3.9.1-mod/Modules/Pl
    )
  
  if(CMAKE_SYSROOT_COMPILE)
-   set(_cmake_sysroot_compile "${CMAKE_SYSROOT_COMPILE}")
- else()
-   set(_cmake_sysroot_compile "${CMAKE_SYSROOT}")
- endif()
- 
- # Default per-language values.  These may be later replaced after
- # parsing the implicit directory information from compiler output.
- set(_CMAKE_C_IMPLICIT_INCLUDE_DIRECTORIES_INIT
-   ${CMAKE_C_IMPLICIT_INCLUDE_DIRECTORIES}
--  "${_cmake_sysroot_compile}/usr/include"
-+  @libc_dev@/include
-   )
- set(_CMAKE_CXX_IMPLICIT_INCLUDE_DIRECTORIES_INIT
-   ${CMAKE_CXX_IMPLICIT_INCLUDE_DIRECTORIES}
--  "${_cmake_sysroot_compile}/usr/include"
-+  @libc_dev@/include
-   )
- set(_CMAKE_CUDA_IMPLICIT_INCLUDE_DIRECTORIES_INIT
-   ${CMAKE_CUDA_IMPLICIT_INCLUDE_DIRECTORIES}
--  "${_cmake_sysroot_compile}/usr/include"
-+  @libc_dev@/include
-   )
```

Comment:
This definitely still looks relevant:

https://github.com/Kitware/CMake/blob/2d0fe11f3e65bdcda3c76c4969c0e4f6e5e2ae75/Modules/Platform/UnixPaths.cmake#L78-L89

Looks like the patch did not apply because of change to the comment on line 65 but this path remained unaffected.

Reply:
Thank you, this should be fixed now :)